### PR TITLE
NATI covers several gx fact types

### PR DESCRIPTION
**Pull request by tychonievich**
Update five terms that are subsumed by NATI:

- Clan
- Ethnicity
- Heimat
- Nationality
- Tribe

Note: I'm unsure of "Heimat" as it is not a term used in my native culture, but the references I could find online seemed to indicate it was subsumed by NATI.

See also the partially-related #314 which proposes changing the one-word title of the NATI structure type in the spec to better match its definition.

### CODE REVIEW

**Review by dthaler — APPROVED** [review]

**Comment by dthaler** [comment]
Discussion 6/29/2023:
Heimat always refers to a place, not an adjective for a person from a place or lineage.
By analogy, an American's Heimat might be "America" or "Washington" or "Seattle", not "American" or "Washingtonian" or "Seattlite".  Would NATI be used with place names, or only the latter type of adjectives?  If the latter, then Heimat does not belong in this set.

File: attribute-event-requests/INDI-attribute.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ Guides for using this document can be found in the associated [README.md](README
...
 | `CAST` | 5.0 | [Caste](#caste) | |
 | \*     | 5.0 | [Childless](#childless) | encoded as `NCH 0` |
-| | | [Clan](#clan) | |
+| `NATI` | 5.4 | [Clan](#clan) | one of several types of `NATI` |
 | \* | 3.0 | [Died Before Eight](#died-before-eight) | encoded as `DEAT`.`AGE <8y` |
 | `EDUC` | 4.0 | Education | |
-| | | [Ethnicity](#ethnicity) | |
-| | | [Heimat](#heimat) | |
+| `NATI` | 5.4 | [Ethnicity](#ethnicity) | one of several types of `NATI` |
+| `NATI` | 5.4 | [Heimat](#heimat) | one of several types of `NATI` |
```
Comment (tychonievich) [inline attribute-event-requests/INDI-attribute.md:17]:
```suggestion
| | | [Heimat](#heimat) | |
```

File: attribute-event-requests/INDI-attribute.md [hunk 2]
```diff
@@ -180,11 +182,12 @@ Found in the following historical records:
 
 The most closely related structures are:
 
-- 
+- `NATI`: Defined as "an individual’s national heritage or origin, or other folk, house, kindred, lineage, or tribal interest." While "heimat" is not specifically included, it is related to "origin" and "house" which are.
```
Comment (tychonievich) [inline attribute-event-requests/INDI-attribute.md:185]:
```suggestion
- `NATI`: Defined as "an individual’s national heritage or origin, or other folk, house, kindred, lineage, or tribal interest." By contrast "heimat" refers to a geographical region a person is from. That could overlap with a national or other regional identity, but need not do so in general.
```

File: attribute-event-requests/INDI-attribute.md [hunk 3]
```diff
@@ -180,11 +182,12 @@ Found in the following historical records:
...
 
 Related proposals include
 
 - Ethnicity: ethnicities typically have some geographic association, but generally much larger and less fine-grained than a heimat
+- Nationality: generally broader than heimat in what region it covers
```
Comment (tychonievich) [inline attribute-event-requests/INDI-attribute.md:190]:
```suggestion
- Nationality: nations are often larger regions than heimats, and may refer to nations that a person belongs to but is not geographically from
```

**Review by tychonievich — COMMENTED** [review]
Changes to remove heimat from the NATI family

**Comment by Norwegian-Sardines** [comment]
This can probably be closed, but I’m still not convinced that Heimat should not be part of this concept.  My understanding is that it is not equivalent to the place but that it is indeed the association to that place, one “homeland”!  The origin of the term as described in https://en.wikipedia.org/wiki/Heimat#Origin_of_the_term
I would use this to identify the region in German (and equally in Norway) my ancestors came from!  

This concept may have changed in more modern times as national identity in the world became more prevalent over regional identity the use of this term may have changed.

But that being said I would not use NATI to identify either Seattle or Seattlite. But I would use it to identify a person as Bavarian, or a Farm name, both still places!

**Pull request merged**